### PR TITLE
Improve identity cli ux:

### PR DESCRIPTION
**Pull request by bryanchriswhite**
+ add messaging
+ fix `--identity-dir` flag
+ function naming cleanup

### CODE REVIEW

File: pkg/identity/certificate_authority.go [hunk 1]
```diff
@@ -95,21 +96,33 @@ func NewCA(ctx context.Context, opts NewCAOptions) (_ *FullCertificateAuthority,
...
-	log.Printf("Generating a certificate matching a difficulty of %d\n", opts.Difficulty)
+	fmt.Printf("Generating key with a minimum a difficulty of %d...\n", opts.Difficulty)
+	logStatus := func() {
+		count := atomic.LoadUint32(i)
+		hs := atomic.LoadUint32(highscore)
+		fmt.Printf("\rGenerated %d keys; best difficulty so far: %d", count, hs)
+	}
 	err = GenerateKeys(ctx, minimumLoggableDifficulty, int(opts.Concurrency),
 		func(k *ecdsa.PrivateKey, id storj.NodeID) (done bool, err error) {
+			count := atomic.LoadUint32(i)
```
Comment (bryanchriswhite) [inline pkg/identity/certificate_authority.go]: Is this awful for performance?

Comment (egonelbre) [reply pkg/identity/certificate_authority.go]: Use `count := atomic.AddUint32(i, 1)` and it's ~10ns per call.... i.e. the modulo operator can be slower.

Comment (egonelbre) [reply pkg/identity/certificate_authority.go]: * it's lower when there is no contention

Comment (bryanchriswhite) [reply pkg/identity/certificate_authority.go]: ha - that makes sense

**Comment by bryanchriswhite** [comment]
@jtolds yea - i was gonna do that but wanted to make sure i got this far at least by today's test run... will do in another PR.

**Pull request merged**